tools/run_local_ironic.sh

```diff
@@ -132,11 +132,8 @@ sudo mkdir -p "$IRONIC_DATA_DIR/html/images"
 # By default, image directory points to dir having needed images when metal3-dev-env environment in use.
 # In other cases user has to store images beforehand.
 
-for name in ironic ironic-api ironic-conductor ironic-inspector dnsmasq httpd mariadb ipa-downloader \
-    ironic-endpoint-keepalived ironic-log-watch ironic-inspector-log-watch; do
-    sudo "${CONTAINER_RUNTIME}" ps | grep -w "$name$" && sudo "${CONTAINER_RUNTIME}" kill "$name"
-    sudo "${CONTAINER_RUNTIME}" ps --all | grep -w "$name$" && sudo "${CONTAINER_RUNTIME}" rm "$name" -f
-done
+# shellcheck disable=SC1091
+source "tools/remove_local_ironic.sh"
```

Comment:
no need to source, you can just run the script, it would remove the shellcheck issue altogether I think

Reply:
I was sourcing because I wanted to retain the env vars of parent shell